Makefile

```diff
@@ -155,3 +155,11 @@ python-dev-setup:
 
 # activate venv
 	@echo "run 'source ./venv/bin/activate' to activate venv in local shell"
+
+# re-generate docs
+.PHONY: clean-changelog
+clean-changelog:
+	rm CHANGELOG.md
+
+.PHONY: generate-changelog
+generate-changelog: clean-changelog $(PWD)/CHANGELOG.md
```

Comment:
is the recipe to be filled in later?

Reply:
No, left empty on purpose and just leveraging dependent targets to force the behavior.

 We have the target `$(PWD)/CHANGELOG.md` but the only way to run it today is part of the larger `generate` or `generate-fast`. I wanted a way to quickly make changes. For example, when needing to fix a merge conflict. In this scenario, I know that all I need to do is trigger this single step to run. 

So, `clean-changelog` is a new target that just removes the file. Not sure if there is a better way to do that. Then added `$(PWD)/CHANGELOG.md` target to re-create it.